stable/enterprise/values.yaml

```diff
@@ -532,6 +535,19 @@ anchoreConfig:
     ## The value should be a number followed by "w", "d", or "h" to represent weeks, days or hours
     async_execution_timeout: 48h
 
+    ## @param anchoreConfig.reports.cycle_timers.reports_scheduled_queries Interval  in seconds to check for scheduled queries that need to be run
+    ##
+    cycle_timers:
+      reports_scheduled_queries: 600
+
+    ## @param anchoreConfig.reports.use_volume Configure the reports worker to buffer report generation to disk instead of in memory
```

Comment:
```suggestion
    ## @param anchoreConfig.reports.use_volume Configure the reports service to buffer report generation to disk instead of in memory
```